api/datastore/sql/migrations/15_alter_apps.go

```diff
@@ -0,0 +1,84 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/fnproject/fn/api/datastore/sql/migratex"
+	"github.com/jmoiron/sqlx"
+)
+
+func up15(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, "ALTER TABLE apps RENAME TO old_apps;")
+	if err != nil {
+		return err
+	}
+	newTable := `CREATE TABLE apps (
+	id varchar(256) NOT NULL PRIMARY KEY,
+	name varchar(256) NOT NULL UNIQUE,
+	config text NOT NULL,
+	annotations text NOT NULL,
+	created_at varchar(256),
+	updated_at varchar(256),
+	syslog_url text
+);`
+	_, err = tx.ExecContext(ctx, newTable)
+	if err != nil {
+		return err
+	}
+	insertQuery := `INSERT INTO apps(id,name,config,annotations,created_at,updated_at,syslog_url)
+					SELECT  id,name,config,annotations,created_at,updated_at,syslog_url FROM old_apps;`
+
+	_, err = tx.ExecContext(ctx, insertQuery)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Worth a count comparison? make sure everythings made it? Probably not that big a deal really.

Reply:
Pretty sure we'd get an error if we had a problem.